benchmark/static/index.ts

```diff
@@ -1 +1 @@
-import './static.js';
+import './static.tsx';
```

Comment:
These were correct, it should have a `.js` extension, since after transpilation `static.tsx` is written to `static.js`. See last point in https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-make-my-typescript-project-output-esm.